relay-metrics/src/aggregation.rs

```diff
@@ -1076,16 +1087,19 @@ impl Actor for Aggregator {
 /// A message containing a list of [`Metric`]s to be inserted into the aggregator.
 #[derive(Debug)]
 pub struct InsertMetrics {
+    /// The project key
+    pub public_key: ProjectKey,
```

Comment:
Instead of giving `InsertMetrics` and `MergeBuckets` a public key, it would be better to create an alternate version of these messages outside of the `relay-metrics` crate. The aggregator (who is the main implementor) does not need these values. We only re-implemented a handler for these messages outside of the crate since we could adhere to the same interface.

Reply:
done